src/CONST/index.js

```diff
@@ -216,12 +216,16 @@ const CONST = {
         ACTIONS: {
             LIMIT: 50,
             TYPE: {
-                IOU: 'IOU',
                 ADDCOMMENT: 'ADDCOMMENT',
+                CLOSED: 'CLOSED',
                 CREATED: 'CREATED',
+                IOU: 'IOU',
                 RENAMED: 'RENAMED',
             },
         },
+        ARCHIVE_REASON: {
+            MANUALLY_ARCHIVED: 'manuallyArchived',
+        },
```

Comment:
Rather than create this constant, since we don't really need the other archiveReasons to be constants, how about just calling this `manuallyArchived` case `default`?  The actual description doesn't really mention anything about this being a manual action.